libraries/EmbeddedChessPieces/src/ECPChessLogic/ECPChessPieces/ECPBishop.cpp

```diff
@@ -0,0 +1,20 @@
+#include "ECPBishop.h"
+
+ECPBishop::ECPBishop(Dezibot &d, ECPChessField initialField, bool isWhite)
+    : ECPChessPiece(d, initialField, isWhite) {};
```

Comment:
It should be checked if the `initialField` is an official starting position for a chess piece.
The color of the piece can be inferred.

Reply:
I actually had this in a previous version but decided to change it. The reason behind this was when writing the test cases, I didn't want the pieces to start on the "correct" initial field on the board. I needed that flexibility and I assume that this could come in handy later. \:)

Reply:
While I don't think you should adapt the code to make testing easier - after all, that's what mocks are for - it is useful for recreating a chessboard at a later time.